randovania/network_common/game_connection_status.py

```diff
@@ -11,6 +11,8 @@ class GameConnectionStatus(Enum):
     Disconnected = "disconnected"
     TitleScreen = "title-screen"
     InGame = "in-game"
+    Unclaimed = "unclaimed"
+    Empty = "empty"
```

Comment:
Where is this used?

Reply:
https://github.com/randovania/randovania/pull/7490/files#diff-4660f7803bfa7c1d3aa65e7e92d48cfb09a701fc3afda26fffb30a89261a4c93R83-R86

Reply:
And what sets something to empty?

Reply:
anything in the "any worlds" group